pyproject.toml

```diff
@@ -38,8 +38,8 @@ dependencies = [
     "jupyter_core>=4.11.0",
     "jupyter_server>=1.18,<2.0.0",
     "jupyterlab_server>=2.3.0,<3",
-    "nbclient>=0.4.0,<0.7",
-    "nbconvert>=6.4.5,<7",
+    "nbclient>=0.4.0,<0.6",
```

Comment:
```suggestion
    "nbclient>=0.4.0,<0.7",
```

Reply:
Issue while rebasing

Reply:
nbclient 0.7.0 is out. Please make it compatible before releasing voila 0.4

Reply:
Thanks for the heads up!